frontend/projects/upgrade/src/app/core/meta-reducers/clear-state.reducer.spec.ts

```diff
@@ -10,6 +10,13 @@ describe('clearState', () => {
     settings: {
       theme: ThemeOptions.DARK_THEME,
     },
+    auth: {
+      isLoggedIn: false,
+      isAuthenticating: false,
+      user: null,
+      googleCredential: null,
+      redirectUrl: '/home',
+    },
```

Comment:
Do we need to set any attributes (besides `redirectUrl`) here, to test that they get set to the default values when `clearState` is called? The other two attributes in 'settings', for instance, aren't set here. If we do set them, it might be a more definitive test if they're initially set to different values.

Reply:
i updated the specs to make this a little more explicitly tested